corehq/apps/es/tests/test_sms.py

```diff
@@ -0,0 +1,35 @@
+from django.test.testcases import SimpleTestCase
+
+from corehq.apps.es.sms import SMSES
+from corehq.apps.es.tests.utils import ElasticTestMixin
+from corehq.elastic import SIZE_LIMIT
+
+
+class TestSMSES(ElasticTestMixin, SimpleTestCase):
+    def test_processed_or_incoming(self):
+        json_output = {
+            "query": {
+                "filtered": {
+                    "filter": {
+                        "and": [
+                            {"term": {"domain.exact": "demo"}},
+                            {
+                                "not": {
+                                    "and": (
+                                        {"term": {"direction": "o"}},
```

Comment:
i noticed this in the filter too, that direction is lowercase o instead of capital O. is that because ES is doing case-insensitive matching?